functests/1_performance/performance.go

```diff
@@ -340,8 +342,134 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("cannot find MachineConfig object %s", initialKey.Name))
 		})
 	})
+
+	Context("Verify API Conversions", func() {
+		It("[test_id:35887] Verifies v1 <-> v1alpha1 conversions", func() {
+
+			By("Checking v1 -> v1alpha1 conversion")
+			v1alpha1Profile := &performancev1alpha1.PerformanceProfile{}
+			key := types.NamespacedName{
+				Name:      profile.Name,
+				Namespace: profile.Namespace,
+			}
+
+			err := testclient.GetWithRetry(context.TODO(), key, v1alpha1Profile)
+			ExpectWithOffset(1, err).ToNot(HaveOccurred(), "Failed getting v1alpha1Profile")
+			Expect(verifyV1alpha1Conversion(v1alpha1Profile, profile)).ToNot(HaveOccurred())
+
+			By("Checking v1alpha1 -> v1 conversion")
+			v1alpha1Profile.Name = "v1alpha"
+			v1alpha1Profile.ResourceVersion = ""
+			Expect(testclient.Client.Create(context.TODO(), v1alpha1Profile)).ToNot(HaveOccurred())
+
+			v1Profile := &performancev1.PerformanceProfile{}
+			key.Name = v1alpha1Profile.Name
```

Comment:
same, I'd create a new key here

Reply:
What would be the point? The test shows elegantly (I think) how the profile can change faces.

Reply:
well, `key` is a throwaway object anyway:

  type NamespacedName struct {
  	Namespace string
  	Name      string
  }

And I have a bias about being as explicit as possible, especially in tests. But again, not a big deal.

Reply:
done